dev/fattest.simplicity/src/componenttest/topology/impl/LogMonitor.java

```diff
@@ -247,15 +248,22 @@ protected String waitForStringInLogUsingMark(String regexp, long intendedTimeout
      * and verify that the regex does not show up in the logs during the
      * specfied duration.
      *
-     * @param timeout Timeout (in milliseconds)
+     * @param regexp a regular expression to search for
+     * @param intendedTimeout a timeout, in milliseconds, within which the wait should complete. Exceeding this is a soft fail.
+     * @param extendedTimeout a timeout, in milliseconds, within which the wait must complete. Exceeding this is a hard fail.
+     * @param outputFile file to check
      * @return line that matched the regexp
      */
-    public String verifyStringNotInLogUsingMark(String regexToSearchFor, long timeout) {
+    public boolean verifyStringNotInLogUsingMark(String regexp, long timeout) {
```

Comment:
I disagree with this change. I deliberately changed this method in https://github.com/OpenLiberty/open-liberty/pull/5038 to return a String. I can understand having a method that returns a boolean for this case because we're verifying whether or not a string was in the log - that's clearly best represented by a boolean. But I believe it's arguably more useful to return the actual string that was found in the log because calling methods can then  log or do additional checking on the line that was found. It's been frustrating to me in some cases where I've used this method when it returned a boolean because it didn't tell me what the line was that it matched. Having that information helps me determine if I wrote my regex incorrectly, if the marks in the log haven't been set as I thought, or if it's finding some other line that I wasn't aware would match.